apps/server/src/modules/room/api/dto/request/room-pagination.params.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { PaginationParams } from '@shared/controller';
+import { IsInt } from 'class-validator';
+
+export class RoomPaginationParams extends PaginationParams {
+	@IsInt()
+	@ApiPropertyOptional({ description: 'Page limit, defaults to 10.' })
+	override limit?: number = 50;
```

Comment:
do we plan to remove that override later? I thought we wana go without the limit all together